### DISCUSSION TITLE
4 button Tuya device not working -  No converter available for 'TS004F' with cluster 'genLevelCtrl' and type 'commandStep' and data '{"stepmode":0,"stepsize":51,"transtime":10}'

### DISCUSSION THREAD

**Discussion by adekloet** [Koenkk/zigbee2mqtt#10039] [Devices]
The latest update  - Current version: 1.22.1-1 the 4 button tuya device is not working anymore (no converter available:

No converter available for 'TS004F' with cluster 'genLevelCtrl' and type 'commandStep' and data '{"stepmode":0,"stepsize":51,"transtime":10}'

It worked in the last version.

Anybody have an idea about this device why we lost support for the 4 button TS004F Tuya?

**Comment by adekloet** [comment]
Device type EndDevice
Zigbee Model TS004F
Zigbee Manufacturer _TZ3000_xabckq1v
Description  Wireless switch with 4 buttons

**Comment by MarcelJ28** [comment]
Same problem here. Until version 1.22.0-3 id did work When I update to version 1.22.1 there is nothing to see at the action but the lqi does change when I press a button. Also the battery percentage is shown. When I go back to 1.22.0-3 I have to pair the switches again and then the are working fine again.

**Comment by adekloet** [comment]
@MarcelJ28 Can you install the blueprint for the switch and give that a try? I repaired the switch and installed the blueprint. The blueprint seems to be listening to certain commands. The difference was that the button layout was slightly different so I tested with just one lamp and assigned it in the blueprint to a button short press and that way identified which button was 1 to 4. You can find the blueprint in the repository: Stringer/zigbee2mqtt-tuya-4-button-scene-switch-ts0044.yaml  It might work for you to.

**Reply by MarcelJ28** [reply]
Well I do not know how to do that. And I am not sure it will solve my problems because I use Node-red for my automations....
